config/api.yml

```diff
@@ -477,10 +477,17 @@
       :post:
       - :name: query
         :identifier: cloud_volume_show_list
+      - :name: delete
```

Comment:
I see there's a test below but I don't think it covers the bulk delete you've added here - can you add one?